zerver/lib/url_redirects.py

```diff
@@ -21,6 +21,10 @@ class URLRedirect:
 
 HELP_DOCUMENTATION_REDIRECTS: List[URLRedirect] = [
     # Add URL redirects for help center documentation here:
+    URLRedirect("/help/pm-mention-alert-notifications", "/help/dm-mention-alert-notifications"),
+    URLRedirect("/help/restrict-private-messages", "/help/restrict-direct-messages"),
+    URLRedirect("/help/reading-pms", "/help/reading-dms"),
```

Comment:
@alya do we want to take the opportunity to make this "Reading direct messages" rather than "Reading DMs" (and thus also make the URL not have the abbreviation, here)? Similar question for `/help/dm-mention-alert-notifications`